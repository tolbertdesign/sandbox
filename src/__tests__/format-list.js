const formatList = require('../lib/format-list')

test('can format a list', () => {
  const formattedList = formatList(
    'Star Wars Names',
    [
      {name: 'Qui-Gon Jinn'},
      {name: 'Chewbacca'},
      {name: 'Han Solo'},
      {name: 'Luke Skywalker'},
    ],
    'name',
  )
  expect(formattedList).toMatchSnapshot()
})
