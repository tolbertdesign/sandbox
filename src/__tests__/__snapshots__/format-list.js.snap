// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can format a list 1`] = `
"These are the items in the Star Wars Names:
  - Qui-Gon Jinn
  - Chewbacca
  - Han Solo
  - Luke Skywalker"
`;
